MuonAnalysis/MomentumScaleCalibration/plugins/MuScleFitMuonSelector.cc

```diff
@@ -217,7 +217,9 @@ void MuScleFitMuonSelector::selectMuons(const edm::Event & event, std::vector<Mu
   if( speedup_ == false )
   {
     if( PATmuons_ ) {
```

Comment:
I meant L219. Because it all uses the same function no matter it is PAT or not?
selectGenSimMuons(event, genPair, simPair, plotter);